eng/tools/BaselineGenerator/Program.cs

```diff
@@ -212,6 +213,7 @@ private async Task<int> Run()
                     NewLineOnAttributes = false,
                     OmitXmlDeclaration = true,
                     WriteEndDocumentOnClose = true,
+                    NewLineChars = "\n", // The file uses LF, not CRLF!
```

Comment:
I didn't end up needing to use this tool but when I first tried it, it changed every newline!